package.json

```diff
@@ -35,11 +35,13 @@
     "memoizerific": "1.8.4",
     "moment": "2.17.1",
     "prerender-node": "2.4.0",
+    "qrcode.react": "^0.6.1",
     "rc-progress": "2.0.3",
     "react": "15.4.1",
     "react-datetime": "2.7.5",
     "react-dom": "15.4.1",
     "react-highcharts": "11.0.0",
+    "react-modal-dialog": "^4.0.7",
```

Comment:
Prefer fixed package version numbers (no leading `^`) for dependencies -- pushing now.